internal/pkg/template/templates/workloads/partials/cf/workload-container.yml

```diff
@@ -48,6 +48,11 @@
     StartPeriod: {{.HealthCheck.StartPeriod}}
     Timeout: {{.HealthCheck.Timeout}}
 {{- end}}
+{{- if .Storage}}
+{{- if .Storage.ReadonlyRootFS}}
+  ReadonlyRootFilesystem: {{.Storage.ReadonlyRootFS}}
```

Comment:
Is `ReadonlyRootFS` has default value set to true? Because the [issue](https://github.com/aws/copilot-cli/issues/4010) says we should by default have following config in the manifest. 
```
storage:
  readonly_root_fs: true
```

Reply:
This cloudformation template looks good to me! (only enable the feature if `.Storage.ReadonlyRootFS` is true).

But 👍 to Parag's feedback. We would need to add the field in the manifest in these files:
https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/template/templates/workloads/services/lb-web/manifest.yml

So that the default generated manifest has the field set to true

Reply:
working on  it